packages/react-jss/src/types.js

```diff
@@ -18,15 +18,16 @@ export type InnerProps = {
 }
 // Needs to be hard coded for stricter types
 export type Context = {
-  '64a55d578f856d258dc345b094a2a2b3'?: Jss,
-  d4bd0baacbc52bbd48bbb9eb24344ecd?: SheetsRegistry,
-  b768b78919504fba9de2c03545c5cd3a?: {[key: number]: SheetsManager},
-  '6fc570d6bd61383819d0f9e7407c452d': StyleSheetFactoryOptions & {disableStylesGeneration?: boolean}
+  jss?: Jss,
+  registry?: SheetsRegistry,
+  managers?: {[key: number]: SheetsManager},
```

Comment:
we should use the same type in other places where we use managers

Reply:
We do.